Source/ProgressionSystemRuntime/Public/LevelActors/PSStarActor.h

```diff
@@ -9,21 +9,31 @@ UCLASS()
 class PROGRESSIONSYSTEMRUNTIME_API APSStarActor : public AActor
 {
 	GENERATED_BODY()
-	
-public:	
+
+public:
 	// Sets default values for this actor's properties
 	APSStarActor();
 
-	UFUNCTION()
-	void SetStaticMesh(UStaticMesh* Mesh);
-	
 protected:
 	// Called when the game starts or when spawned
 	virtual void BeginPlay() override;
-	UPROPERTY()
-	TObjectPtr<class UStaticMeshComponent> MeshComponentInternal = nullptr;
 
-public:
+	/** Function called every frame on this Actor */
+	virtual void Tick(float DeltaTime) override;
+
+	/** Stores the starting time to hide stars */
+	UPROPERTY(VisibleInstanceOnly, BlueprintReadWrite, Transient, Category = "C++", meta = (BlueprintProtected, DisplayName = "Starting time to hide stars"))
+	float StartTimeHideStarsInternal = 0.0f;
+
+	/** When a local character load finished */
+	UFUNCTION(BlueprintCallable, Category= "C++", meta = (BlueprintProtected))
+	void OnLocalCharacterReady(APlayerCharacter* Character, int32 CharacterID);
+
+	/** When a cinematicStarted */
```

Comment:
Wrong description here and in .cpp

Reply:
improved comments 
 with https://github.com/h4rdmol/ProgressionSystem/pull/12/commits/7c26ce9bd69bde78932c3b67cf9ef1a332c48922